content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,329 @@
+ 
+.. math::
+    e^{-i Im(k) \delta} = \frac{1}{e}
+ 
+.. math::
+    \delta = \sqrt{ \frac{2}{\omega \mu \sigma}} \simeq \frac{500}{\sqrt{\sigma f}}
+    :label: Skin Depth
+
+We see the skin depth is highly dependent on both the frequency of our signal and the conductivity of the Earth material. In air , the conductivity is almost 0, so we do not notice important decreased of the electromagnetic wave. In the ground, this is different.
+ - the more conductive, the faster the decay is. MT can see very deep in resistive environment.
+ - The lower the frequency, the slower the decay is. Lowest frequencies sample the deepest structures while high frequencies bring information on shallower structures.
+
+ In :numref:`SkinDepth_MT` and in the movie, we can see that even at very high frequency (20000 Hz), MT is still a deep exploration method in resistive environment (:math:`10^{-5} S/m`) with a skin depth of about 1125m. Skin Depth is often use as an estimator for the depth of investigation of a survey.
+
+Reflection and Transmission Coefficients
+****************************************
+
+.. figure:: images/Reflection_MT.png
+ :align: right
+ :scale: 50% 
+ :name: Reflection_MT
+
+ Reflection at 
+ interface
+
+.. figure :: images/Reflection_Efield.png 
+ :align: right
+ :scale: 50% 
+ :name: Reflection_Efield
+
+ Notations for 
+ reflection
+
+
+
+Let define at the j-th interface :math:`(E^i ; H^i)` as the incident waves, :math:`(E^r ; H^r)` as the refleted wave and :math:`(E^t ; H^t)` as the transmitted wave into the ground (:numref:`Reflection_Efield`) 
+
+Using the :ref:`interface conditions<interface_conditions>` for the tangential components of the electric, we can write:
+
+.. math::
+    E^i + E^r = E^t
+    :label: Continuity of E
+
+And same for the magnetic fields
+
+.. math::
+    H^i + H^r = H^t
+    :label: Continuity of H
+
+Using :ref:`Faraday's law<faraday>`, assuming variations in :math:`\mu` are negligible, we also obtain from equation :eq:`Continuity of H` :
+
+.. math::
+    k_j E^i - k_j E^r = k_{j+1} E^t
+    :label: faraday continuity condition
+
+Replacing the differents components of equation :eq:`faraday continuity condition` with equation :eq:`Continuity of E`, we obtain the reflection coefficient R and the transmission coefficient T:
+
+.. math::
+    R = \frac{E^r}{E^i} = \frac{k_j - k_{j+1}}{k_j + k_{j+1}}
+    :label: Reflection Coefficient
+
+.. math::
+    T = \frac{E^t}{E^i} = \frac{2 k_j}{k_j + k_{j+1}}
+    :label: Transmission Coefficient
+
+
+Refraction angle
+****************
+
+
+Field Acquisition
+-----------------
+
+In MT, the source is unknown but we are avoiding the problem by measuring the ratio of the fields. usually at the surface. We define an apparent impedance
+
+.. math::
+    \hat{Z_{xy}} = \frac{E_x}{H_y}
+    :label: Apparent Impedance Definition
+
+
+Notice this is a complex number, with a norm and an angle.
+
+Impendance tensor
+*****************
+
+
+Data
```

Comment:
I think we will want to parse this out into a different file (but if you want to develop it here, no problem)